tests/virt_autotest/virsh_external_snapshot.pm

```diff
@@ -89,4 +89,10 @@ sub run_test {
     }
 }
 
+sub post_fail_hook {
+    my $self = shift;
+
+    $self->SUPER::post_fail_hook;
```

Comment:
@guoxuguang Thanks for the PR. However I want to remind to follow task priority and our sprint planning in future :). This is unplanned and we really hope the 2 virtual network fixes can be done in this sprint with your limited effort. Now the two network task are still open, hope you can concentrate on them and make them delivered. 

As regard to this PR changes, since virsh_external_snapshot.pm  and virsh_internal_snapshot.pm  do not have their own post_fail_hook, then when they fail, they will directly call their parent's post_fail_hook. So IMHO, this PR can be closed.